topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+
+The key difference between the rooted and the unrooted tree is that only the rooted tree shows the **direction** and sequence of branching events. The unrooted tree does not tell us, for example, whether *M. bovis* diverged early or late in the history of the MTBC. It is thus compatible with the old hypothesis that human TB evolved from animal TB. The rooted tree shows that this hypothesis is most likely wrong: animal-associated strains are not ancestral to human-associated strains.
+
+The best way to root a tree is by including an **outgroup**: a species or lineage which we know *a priori* to lay outside the phylogeny we're interested in. *M. canetti* usually serves this purpose for studying the MTBC, but you can also root, for example, a phylogeny of lineage 2 by including a lineage 4 strain.
+
+<img src="./images/mtbc_1strainPerLineage.nwk.COMB.svg" alt="drawing" width="800"/>
+<figcaption align = "left">Fig.1 - Rooted (A) and unrooted (B) phylogeny of the MTBC. Some basic tree vocabulary is shown in red.</figcaption>
+
+
+## Branch lengths
+Besides relatedness and direction, a third important piece of information contained in a phylogeny is the branch length. When a phylogeny was estimated from DNA or protein sequences, branch length usually reflects the evolutionary distance between nodes in the tree. This information can be used to translate distance in terms of expected nucleotide changes into years, and thus to connect evolutionary change to historical events. Molecular dating is not covered in this tutorial, but if you should be interested in this topic, here is a [recent study](https://doi.org/10.1371/journal.ppat.1008067) showing the promises and problems of molecular dating with MTB.
+
+As branch lengths reflect evolutionary distances, they can also be used to identify transmission clusters and outbreaks. Below is a (rooted) tree of the Central Asian Clade (CAC), which is part of lineage 2 [(Eldholm et al. 2016)](https://doi.org/10.1073/pnas.1611283113). The orange color highlights the Afghan strain family within the CAC. At the bottom of the tree, note the clade with short branch lengths. This is how one would expect an outbreak to look in a phylogenetic tree: a set of strains clustering together and separated by extremely short branches, reflecting their almost identical genomes.
+
+<img src="./images/eldholm2016_tree.png" alt="drawing" width="200"/>
+<figcaption align = "left">Fig.2 - Phylogeny of the central Asian clade, including the Oslo outbreak. Modified Eldholm et al. 2016.</figcaption>
+<br/><br/>
```

Comment:
I just removed the alt text, I hope that's fine

Reply:
The alt text is fine! But there's a couple different things together here in this suggestion:

- Please use `![alt-text](image-path "Figcaption text")` instead of manually doing `<img>` and `<figcaption>`
- Alt text is mandatory. And in this case it was good! 
- And notice that I've removed `Fig.2`, because when you use the markdown syntax for referencing images (with our addition of the ` "text"` as part of the path for the figure caption, the figures are numbered for you automatically.

Reply:
Ok, I've changed the figure format now. Thanks for the details!